atoma-bin/atoma_node.rs

```diff
@@ -225,16 +226,17 @@ async fn main() -> Result<()> {
         register_on_proxy(&config.proxy, *node_small_id, &keystore, args.address_index).await?;
     }
 
-    let (compute_shared_secret_sender, compute_shared_secret_receiver) =
+    let (compute_shared_secret_sender, _compute_shared_secret_receiver) =
         tokio::sync::mpsc::unbounded_channel();
 
+    #[cfg(feature = "tdx")]
     spawn_with_shutdown(
```

Comment:
you should keep the `handle` that's returned from the spawn_with_shutdown and put that into the `try_join` at L380.

Reply:
I see, applied changes.